server/src/router/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Router } from "express";
+import {
+  getAllTodos,
+  updateTodo,
+  deleteTodo,
+  addTodo,
+  getSingleTodo,
+} from "../controller/todos";
+
+const router = Router();
+
```

Comment:
These routes should also be protected by the auth middleware

Reply:
These are protected because the full link is localhost:8080/user/todo and im using is authenticated middleware for every user related query. You can check it in "server/src/router/users.ts"